pages/ksphere/kommander/1.1.0-beta/release-notes/index.md

```diff
@@ -11,56 +11,208 @@ enterprise: false
 
 ## Release Notes
 
-### Version v1.1.0-beta.1 - April 22 March 2020
+<!--
+Template:
+
+### Version <VERSION> - <DATE>
 
 | Kubernetes Support | Version |
 | ------------------ | ------- |
 | **Minimum**        | 1.16.0  |
 | **Maximum**        | 1.16.x  |
 | **Default**        | 1.16.8  |
 
-#### Addon Changelog
+#### Features/Improvements
 
-- added podAnnotations to the kommander-ui chart
-- updated ingress values for kubeaddons-catalog
+- <FEATURES/IMPROVEMENTS>
 
-#### Chart Changelog
+#### Bug Fixes
 
-_Used chart version: `0.5.7` -> `0.6.4`_
+- <FIXES>
+- Lots of smaller UX Bugs and Improvements
 
-- create kubecost ns hook
-- dynamically get grafana service url, add readme file
-- hook should not call static service name
+#### Component Versions
 
-##### Chart Subcomponent Versions
+- Addon: ``
+- Chart: ``
+- KCL: ``
+- UI: ``
+- kommander-karma: ``
+- kubeaddons-catalog: ``
+- kommander-thanos: ``
+- grafana: ``
 
-- KCL: [0.5.1 -> 0.5.2](https://github.com/mesosphere/kommander-cluster-lifecycle/compare/v0.5.1...v0.5.2)
-- UI: [3.16.5 -> 3.25.5](https://github.com/mesosphere/kommander/compare/v3.16.5...v3.25.5)
-- kommander-karma: 0.3.9
-- kubeaddons-catalog: 0.1.6
-- kommander-thanos: 0.1.13
-- grafana: 4.5.1
+-->
 
-### Version v1.1.0-beta.0 - April 8 March 2020
+### Version v1.1.0-rc.1 - May 29th 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.16.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+#### Features/Improvements
+
+- Added Access Control functionalities to the UI
+- Added Kubekost integration to the UI
+- Removed namespace suffix from projects and platform service names
+- UI now trims input values to remove leading, trailing and duplicate spaces
```

Comment:
In general, do we like Oxford commas in places like this, @russurquhart1 ?

Reply:
hehe, I guess grammar/typos are generally a thing when I'm writing something 🤷

Reply:
If i see them, then I add them!

Reply:
plus one for Oxford comma!